### PR TITLE
[FLINK-6782][docs] update snapshot documentation to reflect flink 1.3

### PR DESCRIPTION
**Pull request by NicoK**
please also apply to master after accepting

### CODE REVIEW

File: docs/setup/savepoints.md [hunk 1]
```diff
@@ -129,11 +144,11 @@ $ bin/flink savepoint -d :savepointPath
 
 This disposes the savepoint stored in `:savepointPath`.
 
-Note that since savepoints always go to a file system it is possible to also manually delete the savepoint via a regular file system operation. Keep in mind though that the savepoint only stores meta data that points to the actual checkpoint data. Therefore, if you manually want to delete a savepoint, you would have to include the checkpoint files as well. Since there is currently no straight forward way to figure out how a savepoint maps to a checkpoint, it is recommended to use the savepoint tool for this as described above.
+Note that since savepoints always go to a file system it is possible to also manually delete the savepoint via a regular file system operation. Up to Flink 1.2, this way a more tedious task which was solved with this tool.
```
Comment (alpinegizmo) [inline docs/setup/savepoints.md]: Up to Flink 1.2, this was a more tedious task which was performed with the savepoint tool.

**Comment by EronWright** [comment]
Would you mind remarking as to whether it is safe to delete a savepoint after resuming a job from it?

**Comment by NicoK** [comment]
Deleting a savepoint will not affect other savepoints in case you are worried about this.
Other than that it depends on you and what you want to do with it, e.g. keeping it in case you want to resume from it at a later point in case of failures or so.

I did add a few notes which should make the first point clear.

File: docs/setup/savepoints.md [hunk 2]
```diff
@@ -144,7 +144,7 @@ $ bin/flink savepoint -d :savepointPath
 
 This disposes the savepoint stored in `:savepointPath`.
 
-Note that since savepoints always go to a file system it is possible to also manually delete the savepoint via a regular file system operation. Up to Flink 1.2, this way a more tedious task which was performed with the savepoint command above.
+Note that it is possible to also manually delete a savepoint via regular file system operations without affecting other savepoints or checkpoints (recall that each savepoint is self-contained). Up to Flink 1.2, this way a more tedious task which was performed with the savepoint command above.
```
Comment (alpinegizmo) [inline docs/setup/savepoints.md]: ... this was a more tedious ...

**Comment by alpinegizmo** [comment]
I read Eron's request a bit differently. Suppose you've taken a savepoint simply to manage a rescaling or redeployment. How quickly after resuming from the savepoint is it safe to delete it? Should one wait until the new job has been successfully checkpointed, for example?

**Comment by NicoK** [comment]
Personally, I would wait at least for another completed checkpoint before deleting a savepoint but from what I read, I would probably also always keep a savepoint around to recover from. I guess that depends on the availability needs and storage capacity that you have.
Maybe @StefanRRichter or @StephanEwen can elaborate a bit more on best practices there and extend the docs accordingly.

**Comment by zentol** [comment]
+1 to merge.

**Comment by zentol** [comment]
merging.

**Pull request closed without merging**